cmd/rook/osd.go

```diff
@@ -64,7 +64,7 @@ func init() {
 }
 
 func startOSD(cmd *cobra.Command, args []string) error {
-	required := []string{"cluster-name", "mon-endpoints", "mon-secret", "admin-secret", "node-name", "public-ipv4", "private-ipv4"}
+	required := []string{"cluster-name", "cluster-id", "mon-endpoints", "mon-secret", "admin-secret", "node-name", "public-ipv4", "private-ipv4"}
```

Comment:
is having the cluster-id argument a required parameter safe to put in master without the operator work to set that argument in the pod template spec?